ksqldb-parser/src/main/java/io/confluent/ksql/parser/SqlFormatter.java

```diff
@@ -259,7 +259,11 @@ protected Void visitCreateStream(final CreateStream node, final Integer indent)
 
     @Override
     protected Void visitCreateTable(final CreateTable node, final Integer indent) {
-      formatCreate(node, "TABLE");
+      formatCreate(node,
+          node.isSource()
+              ? "SOURCE TABLE"
```

Comment:
We can keep this method as is, and update formatCreate to add the "SOURCE" keyword if the node is a source. Then we won't have to do the same thing for streams.

Reply:
Done